arviz/plots/forestplot.py

```diff
@@ -541,18 +564,32 @@ def treeplot(self, qlist, credible_interval):
             ntiles[0], ntiles[-1] = hpd(values.flatten(), credible_interval)
             yield y, label, ntiles, color
 
-    def ridgeplot(self, mult):
+    def ridgeplot(self, mult, ridgeplot_kind):
         """Get data for each ridgeplot for the variable."""
         xvals, yvals, pdfs, colors = [], [], [], []
         for y, *_, values, color in self.iterator():
             yvals.append(y)
             colors.append(color)
             values = values.flatten()
-            density, lower, upper = _fast_kde(values)
-            xvals.append(np.linspace(lower, upper, len(density)))
+            values = values[np.isfinite(values)]
+
+            if ridgeplot_kind == "auto":
+                kind = "hist" if all(np.mod(values, 1) == 0) else "density"
```

Comment:
this is not a big enough difference for me to care, but figured i'd mention:

![image](https://user-images.githubusercontent.com/2295568/61007596-d3e1e580-a332-11e9-8a23-b808d1fb0e6f.png)

for smaller inputs, the difference is even more negligible.